pkg/proto/core/v1/util.go

```diff
@@ -0,0 +1,201 @@
+package corev1
+
+import (
+	"github.com/rs/zerolog/log"
+	"google.golang.org/protobuf/proto"
+
+	v0 "github.com/authzed/authzed-go/proto/authzed/api/v0"
+)
+
+//revive:disable
+
+// Mnually created to work around the following issue:
+// https://github.com/envoyproxy/protoc-gen-validate/issues/481
+var _Metadata_MetadataMessage_InLookup = map[string]struct{}{
+	"type.googleapis.com/impl.v1.DocComment":       {},
+	"type.googleapis.com/impl.v1.RelationMetadata": {},
+}
+
+//revive:enable
```

Comment:
?

Reply:
Moved to a separate file and updated the lint ignores for the name of the map. Both revive and stylecheck complain about the var name format.